seqr/utils/elasticsearch/constants.py

```diff
@@ -172,6 +173,16 @@
     }
 }
 
+
+def _get_ranks_by_gene(families):
+    ranks_by_gene = defaultdict(list)
+    for record in PhenotypePrioritization.objects.filter(individual__family=families[0]):
```

Comment:
from the ticket: `we should only sort on the top ranked genes, probably the top 100`
So you need to add more filtering here

Reply:
Okay